src/content/contact_us.json

```diff
@@ -0,0 +1,18 @@
+{
+    "sections": [
+        {
+            "title": "Contact Us",
+            "content": [
+                {
+                    "par": "Have a question or want to suggest improvements for our project? We can help."
+                },
+                {
+                    "par": "Go to AnitaB.org Open Source Zulip Chat @https://anitab-org.zulipchat.com and get your doubts cleared in #questions stream"
+                },
+                {
+                    "par": "Each active project has its own stream, #vms, #mentorship-system, #portal and #power-up for discussions"

```

Comment:
@siddhipandare your paragraphs look good to me! here are some phrases/ideas if you want to add/make changes:
- Having a question in your mind? We are here to help! Post your question under #questions stream in our Zulip Chat @https://anitab-org.zulipchat.com 
- Each active project has its own discussion stream such as #vms, #mentorship-system, #portal, and #powerup. 
- AnitaB.org Open Source is growing globally. Any feedback or improvement suggestions for our projects are most welcome.

Reply:
This sounds better! Will do 👍

Reply:
I figured that my tone was not as welcoming. I have changed it a bit. Let me know if it is too much 😓